pkg/api/api.go

```diff
@@ -31,7 +31,21 @@ type Ack struct{}
 type Backend int
 
 const (
-	GrafanaBackend Backend = iota
+	GrafanaBackend Backend = iota + 1
```

Comment:
Why did you change this to be `iota + 1`?
edit: to not conflict with the zero value -- that's fine.